documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2200,6 +2201,62 @@ Contains a structure with two fields:
 
 |===
 
+[[debezium-mysql-connector-converters]]
+== Custom converters
+
+The {prodname} MySQL connector provides several `CustomConverter` implementations specific for MySQL out of the box.
+These custom converters provide alternative mappings for specific data types influenced by configuration.
+To add a `CustomConverter` to the connector, follow the instructions in the link:../development/converters.adoc[Custom Converters documentation].
+
+=== `TINYINT(1)` to Boolean
+
+MySQL reports `BOOLEAN` data types as `TINYINT(1)` during the connector's snapshot.
+This may not be ideal as it will create a difference in the change event schema with the streaming phase.
+To make sure that MySQL emits `BOOLEAN` columns uniformly, the `TinyIntOneToBooleanConverter` can be used.
+
+.Example configuration
+[source]
+----
+converters=tinyint-one-to-boolean
+converters.tinyint-one-to-boolean.type=io.debezium.connector.mysql.converters.TinyIntOneToBooleanConverter
+converters.tinyint-one-to-boolean.selector=.*.MY_TABLE.DATA
+converters.tinyint-one-to-boolean.length.checker=false
+----
+
+In this example, the `selector` and `length.checker` properties are optional.
+When specifying the `length.checker` as `false`, the converter will not explicitly confirm that the `TINYINT` data type conforms to a length of `1`.
+By default, the length is checked.
```

Comment:
```suggestion
By default, the converter checks that `TINYINT` data types conform to a length of `1`.
Set `length.checker` to `false` to prevent the converter from explicitly confirming the length.
```